addons/common/functions/fnc_cachedCall.sqf

```diff
@@ -4,9 +4,9 @@
  * Returns the result of the function and caches it up to a given time or event
  *
  * Arguments:
- * 0: Parameters <ARRAY>
+ * 0: Parameters <ANY>
  * 1: Function <CODE>
- * 2: Namespace to store the cache on <NAMESPACE>
+ * 2: Namespace to store the cache on <NAMESPACE or OBJECT>
```

Comment:
If `FUNC(eraseCache)` can accept `LOCATION` types, I assume this function should too?

Reply:
technically it can accept group, display...
anything setVar works on (not hashmap)

we often use namespace to mean just that but <namespace> is a actually type
maybe just <ANY>

Reply:
changed to 
```
 * Arguments:
 * Input <ARRAY of BOOLs>
 * Index 0 is the least significant bit.
```